boards/amd/acp_6_0_adsp/doc/index.rst

```diff
@@ -0,0 +1,77 @@
+.. zephyr:board:: acp_6_0_adsp
+
+Overview
+********
+
+ACP 6.0 Audio DSP board is based on AMD ACP 6.0 Xtensa Architecture DSP,
+Zephyr OS is ported to run on the HiFi5 DSP.
+
+Hardware
+********
+
+- Board features:
+
+  - RAM & Storage: 1.75 MB HP SRAM / 512 KB IRAM/DRAM
+  - Audio Interfaces:
+
+      - 1 x SP (I2S, PCM),
+      - 1 x BT (I2S, PCM),
+      - 1 x HS(I2S, PCM),
+      - DMIC
+
+System requirements
+*******************
+
+Xtensa Toolchain
+----------------
```

Comment:
This seems to imply that the Xtensa toolchain is a requirement which I don't think is the case judging by the text that follows.

```suggestion
System requirements
*******************

Xtensa Toolchain (optional)
---------------------------
```

That being said, it does not seem possible to build with the Zephyr SDK - maybe SOF module needs updating?? https://github.com/zephyrproject-rtos/sof/tree/zephyr/src/platform
(really not familiar with this so it might be something else)


```
...
In file included from /Users/kartben/zephyrproject/zephyr/include/zephyr/arch/xtensa/arch.h:29,
                 from /Users/kartben/zephyrproject/zephyr/include/zephyr/arch/cpu.h:27,
                 from /Users/kartben/zephyrproject/zephyr/include/zephyr/kernel_includes.h:36,
                 from /Users/kartben/zephyrproject/zephyr/include/zephyr/kernel.h:17,
                 from /Users/kartben/zephyrproject/zephyr/include/zephyr/pm/device.h:11,
                 from /Users/kartben/zephyrproject/zephyr/kernel/include/kernel_offsets.h:7,
                 from /Users/kartben/zephyrproject/zephyr/arch/xtensa/core/offsets/offsets.c:7:
/Users/kartben/zephyrproject/modules/hal/xtensa/include/xtensa/config/core.h:51:10: fatal error: xtensa/config/core-matmap.h: No such file or directory
   51 | #include <xtensa/config/core-matmap.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
...
```

Reply:
https://github.com/zephyrproject-rtos/hal_xtensa/pull/33 this PR might update SOF modules.